control-plane-operator/controllers/hostedcontrolplane/oauth/params.go

```diff
@@ -42,8 +43,8 @@ type OAuthServerParams struct {
 	LoginURLOverride        string
 	AvailabilityProberImage string `json:"availabilityProberImage"`
 	Availability            hyperv1.AvailabilityPolicy
-	Socks5ProxyImage        string
-	NoProxy                 []string
+	ProxyImage              string
```

Comment:
can we // document this new parameters?
should we group together all the proxy related params into their own struct?
that would also alleviate the reconcileDep signature which whereas is good to be explicit it's already a bit out of hand.

Reply:
Thx, updated.